uplc-semantics.md

```diff
@@ -409,6 +412,70 @@ module UPLC-SEMANTICS
 	    #then (True)
 	    #else (False)
 	    #fi) ... </k>
+
+  // sha3_256
+  // UPLC bytestrings are almost what K's Sha3_256 needs. However, a proper string needs to
+  // be built from the ground up. This is done with the following steps:
+  // 1. We convert the given ByteString to a string representing an hex number and then do its
```

Comment:
replace "then do its ..." to "then convert to its decimal representation using `String2Base(trimByteString(B), 16)`"

Reply:
OK